extensions/2.0/Khronos/KHR_framebuffer_value_mapping/README.md

```diff
@@ -0,0 +1,134 @@
+# KHR_framebuffer_value_mapping
+
+## Contributors
+
+Rickard Sahlin, <mailto:rickard.sahlin@inter.ikea.com>  
+
+Copyright (C) 2024 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.01 spec.
+
+## Exclusions
+
+## Termonilogy
+
+This extension uses the following terms:
+
+Framebuffer : One or more memory buffers used by the target platform to store pixel values prior to being output to display.  
+framebuffer_max_value : The maximum value that can be stored in the framebuffer, for an 8 bit unorm buffer this is 1.0.  
+
+
+## Overview
+
+This extension is intended for implementations that targets a light emitting display with the goal of outputting interactive framerates in a physically correct manner.    
+The purpose is to define how light calculation output values are mapped to framebuffer values.  
+
+There are two main goals for this extension:  
+  
+1: Provide a method to linearly scale RGB values to the framebuffer format of the target device - called lightScaleFactor.  
+  
+2: Define how to limit values prior to writing them to framebuffer when the scaled value exceeds the maximum range of the framebuffer - called RGBThreshold.  
+  
+  
+## Calculation of the lightScaleFactor  
+
+The light scale factor is calculated from the extension "maxLight" value.  
+This value defines the light intensity (lm/m2) value that will be mapped to the maximum value of the framebuffer.  
+ 
+  
+float lightScaleFactor = framebuffer_max_value / maxLight  
+ 
+Each calculated RGB pixel value shall be scaled by the lightScaleFactor at some point prior to being output to display.  
+Exactly how this is done is an implementation detail, the normal usecase would be to scale the value in the fragment shader.   
+
+## Limit of RGB pixel values before output to framebuffer  
+    
+If, after applying the lightScaleFactor, RGB values exceed the maximum value of the framebuffer then the RGB triplet must be scaled.  
+The scaling shall be done using the max component value from the RGB triplet, ensuring that color is retained.  
+  
+pseudocode for scaling.  
+vec3 scaledRGB = RGB / max(R, max(G,B));  
+  
+
+## Defining an asset to use KHR_framebuffer_value_mapping
```

Comment:
Can you add example renders comparing your tone mapping function to others, e.g. Linear, ACES, PBR Neutral and demonstrating the pros and cons compared to each? One model that might make for an interesting comparison is my 3D Macbeth color chart [model](https://github.com/google/model-viewer/blob/5476def9c5bf3d65f04524ce1cc2e6bc849751fb/packages/shared-assets/models/MacbethBalls.glb).

Reply:
I can certainly include a couple of images of how glTFs will look using this extension with different maxLight values.